File: 0771-jewels-and-stones/0771-jewels-and-stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        // jewels의 각 letter를 jewel에 저장
        std::vector<char> jewel;
        for (char &c : jewels) {
            jewel.push_back(c);
        }

        // jewel를 각각 stones와 비교하면서 개수 세기
        int num = 0;
        for (char &i : jewel) {
            for (char &j : stones) {
                if (i == j) {
                    num += 1;
                }
            }
        }

        return num;
    }
};

File: 0771-jewels-and-stones/0771-jewels-and-stones.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        # jewels의 각 letter를 jewel에 저장
        jewel = []
        for j in jewels:
            jewel.append(j)

        # jewel를 각각 stones와 비교하면서 개수 세기
        num = 0
        for j in jewel:
            for s in stones:
                if j == s:
                    num += 1
        return num